frontend/src/components/icons/PhSortAscending.vue

```diff
@@ -0,0 +1,10 @@
+
+<template>
+  <svg xmlns="http://www.w3.org/2000/svg" width="1.2em" height="1.2em" viewBox="0 0 256 256"><path fill="currentColor" d="M128 128a8 8 0 0 1-8 8H48a8 8 0 0 1 0-16h72a8 8 0 0 1 8 8M48 72h136a8 8 0 0 0 0-16H48a8 8 0 0 0 0 16m56 112H48a8 8 0 0 0 0 16h56a8 8 0 0 0 0-16m125.66-21.66a8 8 0 0 0-11.32 0L192 188.69V112a8 8 0 0 0-16 0v76.69l-26.34-26.35a8 8 0 0 0-11.32 11.32l40 40a8 8 0 0 0 11.32 0l40-40a8 8 0 0 0 0-11.32"></path></svg>
+</template>
+
+<script>
+export default {
+  name: 'PhSortAscending'
+}
+</script>
```

Comment:
Convert these icon components to composition API. You don't need "export default" then. Check other Icon files in this same folder. Also, rename the Icon to "SortAscendingIcon.vue"? To be uniform with other icon names